imports/plugins/included/payments-stripe-connect/server/Methods/methods.js

```diff
@@ -0,0 +1,27 @@
+import { Meteor } from "meteor/meteor";
```

Comment:
i made this change so not sure why its still on here? also updated to update Shops collection instead of client-only SellerShops